### PR TITLE
Tensor basics math expression fixes

### PR DESCRIPTION
**Pull request by xmyqsh**
fix some math expressions

### CODE REVIEW

**Comment by zackchase** [comment]
@JeanKossaifi - Dr. Jean - please review this PR!

File: chapter16_tensor_methods/tensor_basics.ipynb [hunk 1]
```diff
@@ -483,11 +483,11 @@
...
+    "As a consequence, to get the n-mode product of $\\tilde X$ by $M$, we can simply take a matrix product between $M$ and the unfolding of $\\tilde X$ along the $n^{th}$ dimension, and refold the result into a tensor of shape $(I_1, \\cdots, I_{n-1}, D, I_{n+1}, \\cdots, I_N)$.\n",
     "\n",
     "### Tensor times vector\n",
     "\n",
-    "In that case we are contracting over the $n$-th mode by multiplying it with a vector. Given a tensor $\\tilde X$ of size $(I_1, I_2, \\cdots, I_N)$, and a vector $v$ of size $(I_n)$, the $n$-mode product of $\\tilde X$ by $v$ is written $\\tilde X \\times_n v$ and is of size $(I_k, I_1 \\times \\cdots \\times I_{n-1} \\times I_{n+1} \\cdots \\times I_n)$.\n",
+    "In that case we are contracting over the $n$-th mode by multiplying it with a vector. Given a tensor $\\tilde X$ of size $(I_1, I_2, \\cdots, I_N)$, and a vector $v$ of size $(I_n)$, the $n$-mode product of $\\tilde X$ by $v$ is written $\\tilde X \\times_n v$ and is of size $(1, I_1 \\times \\cdots \\times I_{n-1} \\times I_{n+1} \\cdots \\times I_N)$.\n",
```
Comment (JeanKossaifi) [inline chapter16_tensor_methods/tensor_basics.ipynb]: That last one is one should simply be (I_1 \\times \\cdots \\times I_{n-1} \\times I_{n+1} \\cdots \\times I_N)
(when contracting a tensor with a vector,  the mode along which we contract disappears)

Comment (zackchase) [reply chapter16_tensor_methods/tensor_basics.ipynb]: Hey back in US now so getting the ball rolling. @JeanKossaifi, @xmyqsh what's the status? @JeanKossaifi  are all issues resolved to your satisfaction?

Comment (JeanKossaifi) [reply chapter16_tensor_methods/tensor_basics.ipynb]: All looking good, just this one thing to change (when contracting over a mode with a vector, the corresponding mode disappears). It's minor so you can also merge that and I'll send a small PR fixing it and other minor typos.

**Pull request merged**